<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLinearCut</class>
 <widget class="QDialog" name="DialogLinearCut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Cut</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelChannel">
     <property name="text">
      <string>Channel:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxChannel">
     <item>
      <property name="text">
       <string>Active</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelParameter">
     <property name="text">
      <string>Parameter:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxParameter">
     <item>
      <property name="text">
       <string>Long Integral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Short Integral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PSD Factor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Filtered PSD Factor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Baseline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time of CFD Crossing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CFD Delta (Prev Channel)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Max Value</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Min Value</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time of Max</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time of Min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Half-Rise Value</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time of Half-Rise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Temperature</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Seconds from First Event</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time of Flight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Variable 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Variable 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Variable 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Variable 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Variable 5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelMin">
     <property name="text">
      <string>Cut Min:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelMax">
     <property name="text">
      <string>Cut Max:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Cut Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMin">
     <property name="minimum">
      <double>-250000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>250000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMax">
     <property name="minimum">
      <double>-250000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>750000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxChannel</tabstop>
  <tabstop>comboBoxParameter</tabstop>
  <tabstop>doubleSpinBoxMin</tabstop>
  <tabstop>doubleSpinBoxMax</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogLinearCut</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogLinearCut</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxChannel</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogLinearCut</receiver>
   <slot>onChannelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DialogLinearCut</receiver>
   <slot>onNameChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxParameter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogLinearCut</receiver>
   <slot>onParameterChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onParameterChanged()</slot>
  <slot>onChannelChanged()</slot>
  <slot>onLinearCutChanged()</slot>
  <slot>onNameChanged()</slot>
 </slots>
</ui>
